src/Dfe.Spi.GraphQlApi.Domain/Repository/LoadLearningProvidersRequest.cs

```diff
@@ -0,0 +1,12 @@
+using Dfe.Spi.GraphQlApi.Domain.Search;
+
+namespace Dfe.Spi.GraphQlApi.Domain.Repository
+{
+    public class LoadLearningProvidersRequest : LoadEntitiesRequest<LearningProviderReference>
+    {
+        public LoadLearningProvidersRequest()
+        {
+            EntityName = "LearningProvider";
```

Comment:
The entity names are the names of the types in the models lib, so you can safely use `nameof()`, if you like.

Reply:
Cool. Done